### PR TITLE
chore: upgrade ember to v4.4

### PR DESCRIPTION
**Pull request by ndekeister-us**
### Upgrade to Ember 4 (#196)
- Internal changes using `ember-cli-update` 
- Drop test for `ember-default-with-jquery` scenario (non breaking as the scenario was not launched by the CI).

### CODE REVIEW

File: .ember-cli [hunk 1]
```diff
@@ -5,5 +5,11 @@
...
     Setting `disableAnalytics` to true will prevent any data from being sent.
   */
-  "disableAnalytics": true
+  "disableAnalytics": false,
+
+  /**
+  Setting `isTypeScriptProject` to true will force the blueprint generators to generate TypeScript
+  rather than JavaScript by default, when a TypeScript version of a given blueprint is available.
+  */
+  "isTypeScriptProject": false
```
Comment (BlueCutOfficial) [inline .ember-cli:14]: Is it something we want to keep to `false` in this addon as it uses TypeScript?

File: .ember-cli [hunk 2]
```diff
@@ -5,5 +5,11 @@
 
     Setting `disableAnalytics` to true will prevent any data from being sent.
   */
-  "disableAnalytics": true
+  "disableAnalytics": false,
```
Comment (BlueCutOfficial) [inline .ember-cli:8]: I had never heard about that, now I am curious to know what kind of data we're talking about.

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -20,7 +20,6 @@ jobs:
       node-version: 12
       package-manager: yarn
       ember-try-scenarios: "[
-          'ember-lts-3.20',
```
Comment (BlueCutOfficial) [inline .github/workflows/ci.yml:23]: I vote to keep 3.20 if the tests are still passing, it's not a so old LTS IMO.
(Same for README and ember-try files)

**Review by BlueCutOfficial — COMMENTED** [review]
There is a "Merge" commit oin your stack 🙀

File: .ember-cli (hunk 2 above)
Comment (ndekeister-us) [reply .ember-cli:8]: https://github.com/ember-cli/ember-cli/blob/master/docs/analytics.md

LGTM, + it aligns with other addons in https://github.com/DazzlingFugu

File: .github/workflows/ci.yml (hunk 3 above)
Comment (ndekeister-us) [reply .github/workflows/ci.yml:23]: Updated following our discussion on discord, to keep compatibility and tests for last 3 LTS if possible 🎉

**Review by BlueCutOfficial — APPROVED** [review]

**Pull request merged**